.github/workflows/test-e2e-cypress.yml

```diff
@@ -0,0 +1,145 @@
+name: Test / E2E
+
+on:
+  workflow_call:
+    inputs:
+      browser:
+        required: false
+        type: string
+        default: chrome
+      theme:
+        required: false
+        type: string
+        default: go
+      wpVersion:
+        required: false
+        type: string
+        default: ""
+      phpVersion:
+        required: false
+        type: string
+        default: "8.1"
```

Comment:
Unfortunately, I think the default should still be 7.4 for the moment, in our specific case.

Reply:
I added a test for PHP7.4, but did not change the default.